src/i18n/i18n.tsx

```diff
@@ -0,0 +1,22 @@
+import i18next from 'i18next';
+import ar from '../../translationService/ar.json';
+import en from '../../translationService/en.json';
+
+const resources = {
+  ar: {
+    ar: ar,
+  },
+  en: {
+    en: en,
+  },
+};
```

Comment:
```suggestion
const resources = {
  ar: {
    translation: ar,
  },
  en: {
    translation: en,
  },
};
```
Thanks Parcera  for the implementation.
Based on the documentation we should use a proper nameSpace in this key. They recommend to use translation as a default (It can be anything else) if the project is small
https://www.i18next.com/overview/getting-started#basic-sample